### ISSUE TITLE
Licencing

### ISSUE THREAD

**Issue by epiclapser** [bemyeyes/bemyeyes-ios#178]
Hey,

I'm actually trying to develop for the Android version, however it is inactive. So I want to fork it and work on it separately, someone brought up the issue of licencing. In desperation, since no one has replied to me (not even the earlier contributerd), I want to know what specific licence BeMyEyes is licenced under, and is it different for android.

Thanks in advance!

**Comment by herlanijunior** [comment]
Do you tried sending a email to info@bemyeyes.org ?

**Comment by epiclapser** [comment]
Yeah that's the first thing I did, they just gave an auto-reply telling me to look at the FAQ and of course the FAQ didn't have any specific licence mentioned....
